features/step_definitions/features_steps.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+Given('a feature {string} of {provider}') do |feature_name, provider|
+  @service_plan = provider.service_plans.where(name: "Basic").last
+  feature = @service_plan.service.features.find_or_create_by(name: feature_name, scope: "ServicePlan")
+end
+
+Given('an enabled feature {string} of {provider}') do |feature_name, provider|
```

Comment:
This step is very similar to the one above, but just enables the feature additionally. So they could be merged, making the "enabled" optional. You could also use the same step for "disabled"